testsuite/tests/backtrace/backtrace3.native.reference

```diff
@@ -22,6 +24,43 @@ Called from file "backtrace3.ml", line 7, characters 42-53
 Called from file "backtrace3.ml", line 7, characters 42-53
 Called from file "backtrace3.ml", line 7, characters 42-53
 Called from file "backtrace3.ml", line 11, characters 4-11
-Called from file "backtrace3.ml", line 28, characters 11-23
+Re-raised at file "backtrace3.ml", line 27, characters 41-50
+Called from file "backtrace3.ml", line 45, characters 11-23
+e
+Uncaught exception Backtrace3.Error("e")
+Raised at file "backtrace3.ml", line 7, characters 16-32
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 11, characters 4-11
+Re-raised at file "backtrace3.ml", line 30, characters 41-51
+Called from file "backtrace3.ml", line 45, characters 11-23
+f
+Uncaught exception Backtrace3.Error("f")
+Raised at file "backtrace3.ml", line 7, characters 16-32
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 11, characters 4-11
+Re-raised at file "backtrace3.ml", line 35, characters 45-54
+Called from file "backtrace3.ml", line 45, characters 11-23
+g
+Uncaught exception Backtrace3.Error("g")
+Raised at file "backtrace3.ml", line 7, characters 16-32
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 7, characters 42-53
+Called from file "backtrace3.ml", line 11, characters 4-11
+Re-raised at file "backtrace3.ml", line 38, characters 45-55
+Called from file "backtrace3.ml", line 45, characters 11-23
+Uncaught exception Backtrace3.Error("h")
+Raised at file "backtrace3.ml", line 41, characters 10-17
```

Comment:
I've reproduced the AppVeyor error. It's nothing to do with CRLF (which is not surprising, because that would be very strange for it to affect bytecode only). The character reference is `16-17` for bytecode. A little tweaking has shows that it's precisely returning the location of `x` not `raise x`